src/libstd/sys/unix/ext/net.rs

```diff
@@ -121,6 +127,22 @@ impl SocketAddr {
     }
 
     /// Returns true if and only if the address is unnamed.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::os::unix::net::UnixListener;
+    ///
+    /// let socket = match UnixListener::bind("/tmp/sock") {
```

Comment:
IMHO every single example exhaustively matching is sort of overzealous.

```
if let Ok(socket) = UnixListener::bind("/tmp/sock") {
     // show off the feature
}
```

would be a much better alternative IMO.